SU2_CFD/src/driver_structure.cpp

```diff
@@ -186,7 +186,30 @@ CDriver::CDriver(char* confFile,
         geometry_container[iZone][iInst][iMesh]->MatchInterface(config_container[iZone]);
         geometry_container[iZone][iInst][iMesh]->MatchActuator_Disk(config_container[iZone]);
       }
-
+      
+      /*--- If we have any periodic markers in this calculation, we must
+       match the periodic points found on both sides of the periodic BC.
+       Note that the current implementation requires a 1-to-1 matching of
+       periodic points on the pair of periodic faces after the translation
+       or rotation is taken into account. ---*/
+      
+      if (config_container[iZone]->GetnMarker_Periodic() != 0) {
```

Comment:
This part will create a segmentation fault when using the DG solver.  I think we can add "&& !fem_solver" in the if statement of line 196.